containers/sign/TopLogoSection.module.scss

```
@@ -0,0 +1,18 @@
+.box {
+  margin-bottom: 30px;
+  margin-top: 50px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  .Image {
+    display: flex;
+    flex-direction: column;
+    justify-items: center;
+    align-items: center;
+  }
```

flex 속성도 만들어놓은 mixin 사용해보시면 좋을 것 같아요!

```suggestion
@include Flex(column, center, center)
```
으로 사용해보시면 좋을 것 같아요~!